### ISSUE TITLE
unable to connect?

### ISSUE THREAD

**Issue by ghost** [jornane/ncc#3]
```
[fkooman@fralen-tuxed-net build]$ ./ncc vpn.tuxed.net 80
Trying vpn.tuxed.net...
Connection failed: Network is unreachable
[fkooman@fralen-tuxed-net build]$ 
```

**Comment by jornane** [comment]
Doesn't do DNS resolving yet, use IP address

**Comment by jornane** [comment]
Fixed in f7c958c.
